contracts/FeralfileArtworkV3.sol

```diff
@@ -107,12 +124,16 @@ contract FeralfileExhibitionV3 is ERC721Enumerable, Authorizable, IERC2981 {
         string memory fingerprint,
         string memory title,
         string memory artistName,
-        uint256 editionSize
+        uint256 editionSize,
+        uint256 aeAmount,
+        uint256 ppAmount
     ) private {
         require(bytes(title).length != 0, "title can not be empty");
         require(bytes(artistName).length != 0, "artist can not be empty");
         require(bytes(fingerprint).length != 0, "fingerprint can not be empty");
         require(editionSize > 0, "edition size needs to be at least 1");
+        require(aeAmount > 0, "AE amount needs to be at least 1");
+        require(ppAmount > 0, "PP amount needs to be at least 1");
```

Comment:
Both `aeAmount` and `ppAmount` can be zero. We can remove this check

Reply:
Fixed